lib/index.js

```diff
@@ -37,8 +38,7 @@ var _settings = exports._settings = {
 
   // This is a node-cron cron job. This will re-fetch the configuration on
   // a set interval, based on the cron-job configuration
-  cronJob: null,
-
+  cronJob: null
```

Comment:
The last property in an object doesn't need a trailing comma and it actually shows an error in an IDE like WebStorm.